assembl/static2/css/components/posts.scss

```diff
@@ -215,15 +234,36 @@ $deleted-post-border: 1px solid $light-grey !default;
   }
 
   .post-left {
-    width: 93%;
-    padding: 40px;
+    width: 100%;
+    padding: 20px 10px 20px 30px;
+
+    .meta {
+      display: flex;
+
+      .author .assembl-icon-back-arrow {
+        font-size: 12px;
+        padding-left: 6px;
```

Comment:
you have 6px her, 4px underneath ? Can it be harmonized ?

Reply:
5px for instance everywhere ?

Reply:
Right you are